purchases/src/main/kotlin/com/revenuecat/purchases/storage/DefaultFileRepository.kt

```diff
@@ -39,25 +41,41 @@ internal interface FileRepository {
      * @return The local file's URI where the data can be found after caching is complete.
      */
     suspend fun generateOrGetCachedFileURL(url: URL): URI
+
+    /**
+     * Get the cached file url if it exists.
+     * @param url The url for the remote data to cache into a file.
+     * @return The local file's URI where the data can be found after caching is complete.
+     * */
+    fun getFile(url: URL): URI?
 }
 
 /**
  * The file repository is a service capable of storing data and returning the URL where that stored data exists.
  */
-internal interface LocalFileCache {
+@InternalRevenueCatAPI
+interface LocalFileCache {
     fun generateLocalFilesystemURI(remoteURL: URL): URI?
     fun cachedContentExists(uri: URI): Boolean
     fun saveData(data: ByteArray, uri: URI)
 }
 
-internal class DefaultFileRepository(
+@InternalRevenueCatAPI
+class DefaultFileRepository
+internal constructor(
     @get:VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    internal val store: KeyedDeferredValueStore<URL, URI> = KeyedDeferredValueStore<URL, URI>(),
+    val store: KeyedDeferredValueStore<URL, URI> = urlToUriStore,
```

Comment:
Hmm any reason we had to make this parameter public instead of keeping it internal?

Reply:
that was a mistake I think.